examples/prophet/conda.yaml

```diff
@@ -5,7 +5,7 @@ channels:
   - conda-forge
 dependencies:
   - python=3.6
-  - fbprophet
+  - fbprophet<=0.7
```

Comment:
We could add plotly here instead of pinning `fbprophet`, but our `fbprohet` example ([link](https://github.com/mlflow/mlflow/tree/master/examples/prophet)) does seem to use [the plot APIs](https://github.com/facebook/prophet/blob/9fb8f103d623fc9e3b45dfd895664cc91bc82d2b/python/fbprophet/plot.py)